subset/security/nginx.conf

```diff
@@ -0,0 +1,35 @@
+events {
```

Comment:
remove entire file, I dont think this is required

Reply:
fixed